en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands. (A system user is not required.)
```

Comment:
Yes, removed the parenthetic statement. I suppose there should be another topic that discusses console-conf, the user it creates, that it is optional, and how to suppress it.